<?xml version="1.0" encoding="UTF-8"?>
<!--
SPDX-FileCopyrightText: 2020 Michael Reeves reeves.87@gmail.com
SPDX-License-Identifier: GPL-2.0-or-later
-->
<ui version="4.0">
 <class>OpenDialog</class>
 <widget class="QDialog" name="OpenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="virticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>B:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>A (Base):</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Output (Optional):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="mergeCheckBox">
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="fileSelectC">
       <property name="text">
        <string>File...</string>
       </property>
       <property name="icon">
        <iconset theme="document-new">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="swapCopy">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Swap/Copy Names...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>C (Optional):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="lineA">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="lineC">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="lineB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="fileSelectB">
       <property name="text">
        <string>File...</string>
       </property>
       <property name="icon">
        <iconset theme="document-new">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="folderSelectB">
       <property name="text">
        <string>Folder...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open-folder">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="folderSelectC">
       <property name="text">
        <string>Folder...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open-folder">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="folderSelectA">
       <property name="text">
        <string>Folder...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open-folder">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="fileSelectA">
       <property name="text">
        <string>File...</string>
       </property>
       <property name="icon">
        <iconset theme="document-new">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="selectOutputFile">
       <property name="text">
        <string>File...</string>
       </property>
       <property name="icon">
        <iconset theme="document-new">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="lineOut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="selectOutputFolder">
       <property name="text">
        <string>Folder...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open-folder">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineA</tabstop>
  <tabstop>fileSelectA</tabstop>
  <tabstop>folderSelectA</tabstop>
  <tabstop>lineB</tabstop>
  <tabstop>fileSelectB</tabstop>
  <tabstop>folderSelectB</tabstop>
  <tabstop>lineC</tabstop>
  <tabstop>fileSelectC</tabstop>
  <tabstop>folderSelectC</tabstop>
  <tabstop>mergeCheckBox</tabstop>
  <tabstop>swapCopy</tabstop>
  <tabstop>lineOut</tabstop>
  <tabstop>selectOutputFile</tabstop>
  <tabstop>selectOutputFolder</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
